packages/frontend/src/hooks/organisation/useOrganisation.ts

```diff
@@ -13,4 +13,6 @@ export const useOrganisation = () =>
     useQuery<Organisation, ApiError>({
         queryKey: ['organisation'],
         queryFn: getOrganisation,
+        retry: false,
```

Comment:
Isn't `retry` to do with if there are any errors? Did this affect rendering?

Reply:
No, it does not affect rendering, I added it because we use it on other components, but I'll remove it